pyproject.toml

```diff
@@ -1,20 +1,18 @@
 [build-system]
-
 requires = [
     "setuptools>=42",
-    "wheel>=0.36",
     "oldest-supported-numpy",
 ]
+build-backend = "setuptools.build_meta"
 
-[tool.cibuildwheel]
 
+[tool.cibuildwheel]
 skip = "pp* *musllinux*"
 test-skip = "*i686 *win32 *-macosx_arm64"
-
 before-test = "pip install cython; pip install --extra-index-url https://test.pypi.org/simple energyflow==1.3.3a0"
 test-command = "pytest {package}"
 test-requires = ["pytest", "numpy", "pot", "energyflow"]
 
+#FIXME delete with closure of https://github.com/thaler-lab/Wasserstein/issues/13
```

Comment:
Is this just a comment to self? Or is there a follow up action needed here? If just a comment and nothing needs to get edited in terms of files then maybe just remove this so that this doesn't get forgotten, as the Issue will get closed when resolved.

```suggestion
```

Reply:
@henryiii mentioned in #10 that the custom architecture mentioned here will not be needed once MacOS-14 is supported. I don't know if removing it now will break things.

Reply:
Ah 

> The custom arch stuff in pyproject.toml can then be removed, as native will work fine.

okay. Thanks I had missed that earlier. I think it is probably fine to delete that here too, as when you add in `macos-14` support this won't be needed anyway.

Reply:
Shouldn’t you do this when you add support for macos-14? With this present, either image will build the native architecture, and cross compile the other architecture. Without it, they will only build their own native architectures.

Reply:
Sorry missed this as on mobile. Yes, but that seemed fine to lose cross compiling now as I was going to suggest that once things are building as expected with libomp to overhaul the wheel building CI to basically be a copy of boost-histogram's with modifications and so all of this would get added back there. If I am missing something here let me know.

Reply:
I'll wait on making further changes until tomorrow to see what you two land on here.

Reply:
@j-s-ashley I'd defer to @henryiii here, so leave this in for now. I think this is good to go from here.